gemfiles/4.2.awsv2.0.gemfile

```diff
@@ -2,7 +2,7 @@
 
 source "https://rubygems.org"
 
-gem "sqlite3", "~> 1.3.8", platforms: :ruby
+gem "sqlite3", "~> 1.3.8", :platforms => :ruby
```

Comment:
Which makes sense for a testing library (https://github.com/thoughtbot/appraisal/issues/105#issuecomment-196308393). Thanks!